pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "app"
-version = "3.18.0"
+version = "3.15.0"
```

Comment:
why we revert this version back to 3.15?

Reply:
oh this is due to auto merge. will manually fix it.

Reply:
Gotcha